css/index.css

```diff
@@ -143,4 +143,165 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+/*headear*/
+header nav{
+    display: flex;
+    padding:3% 0;
+    justify-content: space-between;
+    align-items: flex-end;
+ 
+}
+header nav a {
+    color: black;
+    text-decoration: none;
+}
+ 
+.middle-content .boxes .box {
+    background-color:teal ;
+    
+}
+/*box*/
```

Comment:
For your boxes, all of these styles are in the .middle-content .boxes .box.  Which is a parent of each box.  You can use this to your advantage by adding an id or second class instead of removing the .box class from the box.  You can do <div class="box box1"> inherit the styles and just override the color of each box.  It saves a lot of lines of CSS code.